docs/source/integrations/faiss.rst

```diff
@@ -1,14 +1,13 @@
-FAISS
+Faiss
 -----
 
-FAISS v1.8 provides a special conda package that enables a RAFT backend for the Flat, IVF-Flat and IVF-PQ indexes on the GPU. Like the classical FAISS GPU indexes, the RAFT backend also enables interoperability between FAISS CPU indexes, allowing an index to be trained on GPU, searched on CPU, and vice versa.
+Faiss v1.10.0 provides a special conda package that enables a cuVS backend for the Flat, IVF-Flat, IVF-PQ and CAGRA indexes on the GPU. Like the classical Faiss GPU indexes, the cuVS backend also enables interoperability between Faiss CPU indexes, allowing an index to be trained on GPU, searched on CPU, and vice versa.
 
-The RAFT backend can be enabled by building FAISS from source with the `FAISS_USE_RAFT` cmake flag enabled and setting the `use_raft` configuration option for the RAFT-enabled GPU indexes.
+The cuVS backend can be enabled by building Faiss from source with the `FAISS_ENABLE_CUVS` cmake flag enabled and setting the `use_cuvs` configuration option for the cuVS-enabled GPU indexes.
 
 A pre-compiled conda package can also be installed using the following command:
 
-.. code-block:: bash
 
-    conda install -c conda-forge -c pytorch -c rapidsai -c nvidia -c "nvidia/label/cuda-11.8.0" faiss-gpu-raft
+.. code-block:: bash
 
-The next release of FAISS will feature cuVS support as we continue to migrate the vector search algorithms from RAFT to cuVS.
+    conda install -c pytorch -c nvidia -c rapidsai -c conda-forge libnvjitlink faiss-gpu-cuvs=1.10.0
```

Comment:
Now that we have documentation in FAISS for installing faiss-cuvs, we should point there. You should be able to link to the perma-link of this line in the FAISS docs. The reason I'd do that instead of duplicating it is because we'll now need to change this in two places should we ever need to change it. If we just link to the existing install docs then we only need to change it within FAISS itself.